pontoon/translations/views.py

```diff
@@ -23,21 +24,18 @@
 
 
 def _send_badge_notification(user, badge, level):
-    desc = """
-        You have gained a new badge level!
-        <br>
-        {badge}: Level {level}
-        <br>
-        You can view this badge on your <a href={profile_href}> profile page </a>.
-        """.format(
-        badge=badge,
-        level=level,
-        profile_href=reverse(
-            "pontoon.contributors.contributor.username",
-            kwargs={
-                "username": user,
-            },
-        ),
+    desc = render_to_string(
+        "messaging/badge_notification.jinja",
+        {
+            "badge": badge,
+            "level": level,
+            "profile_href": reverse(
+                "pontoon.contributors.contributor.username",
+                kwargs={
+                    "username": user,
+                },
+            ),
```

Comment:
Now that we're using a template, we can construct the URL there (see `suggestion_notification.jinja`), and only pass `user` in context.